packages/sonarwhal/src/lib/connectors/debugging-protocol-common/debugging-protocol-connector.ts

```diff
@@ -673,9 +673,11 @@ export class Connector implements IConnector {
 
         try {
             debug(`resource ${href} to be fetched`);
-            await this._server.emitAsync('fetch::start', { resource: href });
+            const fullFaviconUrl = this._finalHref + href.substr(1);
```

Comment:
The favicon part doesn't seem to be part of this PR?

Reply:
It is part, this PR has some changes in sonarwhal too (in a different commit)

Reply:
OK!